ts/src/oxfun.ts

```diff
@@ -0,0 +1,2463 @@
+
+//  ---------------------------------------------------------------------------
+
+import Exchange from './abstract/oxfun.js';
+import { Precise } from './base/Precise.js';
+import { BadRequest } from './base/errors.js';
+import { TICK_SIZE } from './base/functions/number.js';
+import { sha256 } from './static_dependencies/noble-hashes/sha256.js';
+import type { Account, Balances, Bool, Currencies, Currency, Int, Market, Num, OHLCV, Order, OrderBook, OrderType, OrderSide, Str, Strings, Ticker, Tickers, Trade, Transaction, TransferEntry } from './base/types.js';
+
+//  ---------------------------------------------------------------------------
+
+/**
+ * @class oxfun
+ * @augments Exchange
+ */
+export default class oxfun extends Exchange {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'oxfun',
+            'name': 'Oxfun',
+            'countries': [ 'PA' ], // Panama todo check
+            'version': 'v3',
+            'rateLimit': 120, // 100 requests per second and 25000 per 5 minutes todo check
+            'pro': true,
+            'has': {
+                'CORS': undefined,
+                'spot': true,
+                'margin': false,
+                'swap': true,
+                'future': false,
+                'option': false,
+                'addMargin': false,
+                'cancelAllOrders': false,
```

Comment:
I think we can implement this method
https://docs.ox.fun/?json#delete-v3-orders-cancel-all

Reply:
at work